src/main/java/io/reactivex/Maybe.java

```diff
@@ -82,6 +123,161 @@
     }
     
     /**
+     * Concatenate the single values, in a non-overlapping fashion, of the MaybeSource sources provided by
+     * an Iterable sequence. 
+     * <dl>
+     * <dt><b>Scheduler:</b></dt>
+     * <dd>{@code concat} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     * @param <T> the value type
+     * @param sources the Iterable sequence of MaybeSource instances
+     * @return the new Flowable instance
+     */
+    public static <T> Flowable<T> concat(Iterable<? extends MaybeSource<? extends T>> sources) {
```

Comment:
What's the thinking behind using `Flowable` for all the combination operators vs. `Observable`?